doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -51,7 +51,10 @@ For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Added:
+
+  * Documentation page about :ref:`ug_bt_mesh_fota`.
+  * Documentation page about :ref:`ug_bt_mesh_node_removal`.
```

Comment:
Is node_removal added, or updated?

Reply:
I haven't done any changes in the node_removal page itself. Just moved the release note entry for it to a more fitting place, under our mesh user guide under Protocols.